cd/auto-update/auto-update-controllers.sh

```diff
@@ -18,7 +18,7 @@ Environment variables:
                        be created when auto-update of project description files
                        for service controller fails. Defaults to 'community'
   GITHUB_LABEL:        Label to add to issue and pull requests.
-                       Defaults to 'ack-bot-autogen'
+                       Defaults to 'ack-bot-autoupdate'
```

Comment:
you can leave the github label same as before i.e. `ack-bot-autogen`. I only suggested to update the branch name because it is very similar to auto-gen branch and may cause conflict. Labels do not have that problem.